src/DataCollectors/TraceDataCollector/VanguardCollector/Interfaces/IFileHelper.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.Coverage.Interfaces
+{
+    /// <summary>
+    /// Interface to Abstraction System.IO.File funcationalities for mocking File functionalities
```

Comment:
>funcationalities [](start = 48, length = 16)

Nit: Better summary. Functionalities*
same in IDirectoryHelper

Reply:
fixed.